SensusService/Protocol.cs

```diff
@@ -1012,6 +1121,50 @@ public void Start()
             }
         }
 
+        public void ScheduleStart()
+        {
+            DateTime startTime = new DateTime(_startTime.Year, _startTime.Month, _startTime.Day, _startTime.Hour, _startTime.Minute, 0);
```

Comment:
The user only choose hours and minutes. The documentation doesn't say but I think the time picker sets seconds to the current time's second value I think... Is that ok to use?